pkg/apis/core/types.go

```diff
@@ -5600,6 +5600,31 @@ const (
 	ScheduleAnyway UnsatisfiableConstraintAction = "ScheduleAnyway"
 )
 
+// PolicyName defines the type of node inclusion policy
+type PolicyName string
+
+const (
```

Comment:
I'm a little curious that is there are only two choice here? respect or not respect right？
If so, why not use bool here? `true` or `false` is more easily to use.

Reply:
Use enums here for extensible I supposed so.